tests/sentry/tasks/test_backfill_seer_grouping_records.py

```diff
@@ -1534,3 +1539,55 @@ def test_backfill_seer_grouping_records_killswitch_enabled(self, mock_logger):
         mock_logger.info.assert_called_with(
             "backfill_seer_grouping_records.killswitch_enabled",
         )
+
+    @with_feature("projects:similarity-embeddings-backfill")
+    @patch("sentry.tasks.embeddings_grouping.utils.logger")
+    @patch("sentry.tasks.embeddings_grouping.utils.post_bulk_grouping_records")
+    def test_backfill_seer_grouping_records_enable_ingestion(
+        self, mock_post_bulk_grouping_records, mock_logger
+    ):
+        """
+        Test that the metadata is set for all groups showing that the record has been created.
```

Comment:
Isn't the point of this test (and the next one) to show what happens with the project option, not with the metadata?

Reply:
haha thanks, copy and paste error. I'll fix it